README.md

```diff
@@ -259,5 +259,5 @@ This model is render in [OpenModelica](https://openmodelica.org/) as:
 The flange indicated with the `joint_1_2` label is a classical Modelica acausal flange of type [`Modelica.Mechanics.Rotational.Interfaces.Flange_a`](https://doc.modelica.org/help/Modelica_Mechanics_Rotational_Interfaces.html#Modelica.Mechanics.Rotational.Interfaces.Flange_a), so the model can be interfaced with arbitrary Modelica models.  
 
 ## Naming
-Due to the Modelica naming conventions, the urdf file that is to be translated, should contain model name, link name and joint name with only alphanumeric or underscore characters.
-See also Modelica language specification Appendix B https://www.modelica.org/documents
\ No newline at end of file
+Due to the Modelica naming conventions, the urdf file that is translated, should contain model name, link name and joint name with only alphanumeric or underscore characters.
```

Comment:
Please mention also SDF files and use plurals for link and joint names, as mentioned in https://github.com/robotology-playground/sdf-modelica/pull/12#discussion_r195347740 , thanks!